zaza/openstack/charm_tests/neutron/tests.py

```diff
@@ -592,6 +592,53 @@ def test_901_pause_and_resume(self):
             logging.info('Testing pause resume')
 
 
+class NeutronBridgePortMappingTest(NeutronPluginApiSharedTests):
+    """Test correct handling of network-bridge-port mapping functionality."""
+
+    def test_600_conflict_data_ext_ports(self):
+        """Verify proper handling of conflict between data-port and ext-port.
+
+        Configuring ext-port and data-port at the same time should make the
+        charm to enter "blocked" state. After unsetting ext-port charm should
+        be active again.
+        """
+        if self.application_name in ["neutron-gateway", "neutron-openvswitch"]:
```

Comment:
Rather than just silently ignore the test, why not use `SkipTest()` to indicate that the test was skipped.  e.g.

```python
if self.application_name not in [..]:
    raise unittest.SkipTest("charm under test is not ... so skipping test")
```

This means that the rest of the test doesn't need to be in an if statement.

Reply:
Fixed, thanks. Instead of calling SkipTest() I just return early, same as all other tests in this package.